core/modules/filters/intersects.js

```diff
@@ -0,0 +1,50 @@
+/*\
+title: $:/core/modules/filters/intersects.js
+type: application/javascript
+module-type: filteroperator
+
+Filter operator for comparing the contents of an input tiddlers field and the
+operand. Titles are returned if there is any overlap between the two lists.
+
+\*/
+(function(){
+
+/*jslint node: true, browser: true */
+/*global $tw: false */
+"use strict";
+
+/*
+Export our filter function
+*/
+exports.intersects = function(source,operator,options) {
```

Comment:
What's the difference to contains? This PR really needs a demo to play with!

Reply:
The demo is here http://ooktech.com/jed/ExampleWikis/SetFilters/#contains%20Operator:%5B%5Bcontains%20Operator%5D%5D%20%5B%5Bintersects%20Operator%5D%5D%20%5B%5Bmatches%20Operator%5D%5D